app/views/layouts/ember.html.erb

```diff
@@ -14,13 +14,12 @@
   </head>
   <body class="flex-body" id="application">
 
-     <%= render partial: "shared/analytics" %>
```

Comment:
Can do that as well, though for the ember app I believe the only thing we use it for is the intercom chat widget.